packages/activemq/data_stream/broker/fields/fields.yml

```
@@ -0,0 +1,40 @@
+- name: activemq.broker
+  type: group
+  release: ga
```

Let's start with beta.
```suggestion
  release: beta
```

---

Done